### ISSUE TITLE
Needs more jQuery

### ISSUE THREAD

**Issue by SemiNormal** [dvidsilva/boron.js#3]
This need to be a jQuery plugin. Also, there should be some mongoDB to add extra webscale.

**Comment by gzur** [comment]
I disagree, making this into a plugin would violate the Unix philosophy, there is no reason why this can't be run as a library, since the dependency injection model is sufficiently robust to work alongside any framework you can throw it at.
